nuxt.config.js

```diff
@@ -4,37 +4,41 @@ export default {
    ** Doc: https://vue-meta.nuxtjs.org/api/#metainfo-properties
    */
   head: {
-    title: "Flag Clicked",
+    titleTemplate: (chunk) => {
+      return chunk ? `${chunk} - When Flag Clicked` : "When Flag Clicked";
+    },
```

Comment:
I never knew about this but it looks great... I would like it to say something `How to create a platformer on When Flag Clicked` if it was on a tutorial. That would be pretty cool. Are there also any for the meta stuff? It's fine if this is not in this PR we could open another issue for it.

Reply:
i've been learning nuxt from ocular's source code